airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/java/io/airbyte/cdk/integrations/destination/jdbc/JdbcBufferedConsumerFactory.java

```diff
@@ -147,11 +158,14 @@ private static String getOutputSchema(final AirbyteStream stream,
    */
   private static OnStartFunction onStartFunction(final JdbcDatabase database,
                                                  final SqlOperations sqlOperations,
-                                                 final Collection<WriteConfig> writeConfigs) {
+                                                 final Collection<WriteConfig> writeConfigs,
+                                                 final TyperDeduper typerDeduper) {
     return () -> {
+      typerDeduper.prepareTables();
       LOGGER.info("Preparing raw tables in destination started for {} streams", writeConfigs.size());
       final List<String> queryList = new ArrayList<>();
       for (final WriteConfig writeConfig : writeConfigs) {
+        // These values should be updated for destinations v2
```

Comment:
should we do anything about this? e.g. just remove the schema/table name from the log message :P